packages/ember-data/lib/system/model/model.js

```diff
@@ -295,7 +295,7 @@ var Model = Ember.Object.extend(Ember.Evented, {
     @property id
     @type {String}
   */
-  id: null,
+  id: Ember.computed.alias('_internalModel.id'),
```

Comment:
I'm worried about this being a breaking changed. It is possible to access a record's id without using a getter (`record.id`).  Maybe we want to use an ES5 getter to grab the id out of the internal model?